replication_test.go

```diff
@@ -590,6 +590,10 @@ func sendVotesToOneNode(filteredInNode simplex.NodeID) messageFilter {
 	}
 }
 
+func blockVotesFromRound0(msg *simplex.Message, _, to simplex.NodeID) bool {
```

Comment:
can't we export this into a closure that is defined inside the test? We only use this in one place, and it's only one line.